x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/license_callout/license_callout.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockValues } from '../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiPanel, EuiButton, EuiText } from '@elastic/eui';
+
+import { LicenseCallout } from './';
+
+describe('LicenseCallout', () => {
+  it('renders when non-platinum or on trial', () => {
+    setMockValues({
+      hasPlatinumLicense: false,
+      isTrial: true,
+    });
+    const wrapper = shallow(<LicenseCallout />);
+
+    expect(wrapper.find(EuiPanel)).toHaveLength(1);
+    expect(wrapper.find(EuiText)).toHaveLength(3);
+    expect(wrapper.find(EuiButton).prop('href')).toEqual(
+      '/app/management/stack/license_management'
+    );
+  });
+
+  it('does not render for platinum', () => {
+    setMockValues({
+      hasPlatinumLicense: true,
+      isTrial: false,
+    });
+    const wrapper = shallow(<LicenseCallout />);
+
+    expect(wrapper.find(EuiPanel)).toHaveLength(0);
```

Comment:
[optional] for early `return null`s you can also check if anything has rendered at all with the [isEmptyRender](https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/isEmptyRender.html) API!

```suggestion
    expect(wrapper.isEmptyRender()).toBe(true);
```

Reply:
Thanks!